### DISCUSSION TITLE
Janushgraph with springboot

### DISCUSSION THREAD

**Discussion by Ranjith2002** [JanusGraph/janusgraph#3643] [General]
I need suggestion for connecting janusgraph with springboot

**Comment by porunov** [comment]
> I need suggestion for connecting janusgraph with springboot

Not sure what suggestions you need, but JanusGraph is compatible with SpringBoot in both remote and embedded mode.

**Comment by Ranjith2002** [comment]
I need both method sir. I am in learning phase now. Kindly encourage me
